test/run.d

```diff
@@ -122,7 +133,13 @@ void ensureToolsExists()
             writefln("%s is already up-to-date", tool);
         else
         {
-            auto command = [hostDMD, "-of"~targetBin, sourceFile];
+            auto command = [
+                hostDMD,
+                toolsDir.buildPath("paths"),
```

Comment:
Why not use `-i` here? In either case, I would make a comment that these are the imported dependencies of the tools.
Also, you now compiler `paths.o` into the other tools as well (though this probably is just a few KB more).

Reply:
> Why not use `-i` here?

Does GDC support that flag?

Reply:
Fixed (without using the `-i` flag).